_data/providers.json

```diff
@@ -249,6 +249,11 @@
       "url": "https://humbleservers.com/",
       "description": "Set the Bedrock port to the Java server's port, or to one of the two extra ports, and connect with that port. If the subdomain doesn't work, use your regular numbered IP address."
     },
+    {
+      "name": "KeKsHost",
+      "url": "https://kekshost.com/en/",
+      "description": "Get Geyser as a plugin, as in the setup guide. Use the 19131 port and the server ip and connect with the same IP and port 19132. You can also ask for a new port."
```

Comment:
Now you're asking people to set the port 19131, but connect with 19132? 
This still doesn't seem right. Additionally, please adjust the text flow to make it similar to the other texts (aka, remove the "get geyser as a plugin..." in the beginning)